src/main/kotlin/dev/avt/api/admin/absence/denyHours/DenyHours.kt

```diff
@@ -0,0 +1,58 @@
+package dev.avt.api.admin.absence.denyHours
+
+import dev.avt.database.*
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.auth.*
+import io.ktor.server.request.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import org.jetbrains.exposed.sql.transactions.transaction
+
+fun Routing.denyHoursRoute() {
+    route("api/admin/{personId}/deny_hours") {
+        authenticate("auth-bearer") {
+            post {
+                // TODO test this
```

Comment:
...

Reply:
heb je het getest?

Reply:
Nee tis admin api daar is nog niet echt een goede manier om het te testen voor gemaakt